ui/pages/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -194,28 +213,24 @@ const mapStateToProps = (state, ownProps) => {
   const fromAddressIsLedger = isAddressLedger(state, fromAddress);
   const nativeCurrency = getNativeCurrency(state);
 
-  const hardwareWalletRequiresConnection =
-    doesAddressRequireLedgerHidConnection(state, fromAddress);
-
-  const isMultiLayerFeeNetwork = getIsMultiLayerFeeNetwork(state);
-
   ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-  const accountType = getAccountType(state);
-
   const fromChecksumHexAddress = toChecksumHexAddress(fromAddress);
```

Comment:
above this line we can have the:
`  const accountType = getAccountType(state, fromAddress);`